rr_graph_uxsdcxx.h

```diff
@@ -0,0 +1,398 @@
+#include <bitset>
+#include <cassert>
+#include <cstring>
+#include <iostream>
+#include <limits>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <error.h>
+#include <stddef.h>
+#include <stdint.h>
+#include "pugixml.hpp"
+
+/*
+ * This file is generated by uxsdcxx.
```

Comment:
Please more information on the revision of uxsdcxx (what was the revision of that, where can you get it from, etc?)

This comment should also probably be the first thing in the file?